src/reducers/searchResultsReducer.js

```diff
@@ -69,36 +69,39 @@ export default function course(state = initialState.SearchResults, {payload, typ
     }
 }
 
-const handleAccountSearchResults = (state, payload, status) =>{
-    let {response}= payload;
-    let {data} = response;
-
+const handleAccountSearchResults = (state, payload, status) => {
+    let { response } = payload;
+    let { data } = response;
+    console.log(data)
+    // you can get page and count
     return {
```

Comment:
A better way of structuring the data should be like this:
```
...
account:{
  results: [],
  count: 0,
  page: 1,
 },
course: {
  results: [],
  count: 0,
  page: 1,
}
...
```
In this case we'll need to update this return statement and what we have in initial state to reflect this structure.